portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,60 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <style>
+body {
+  background-image: url('images/white-cat-1.png');
+}
+</style>
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>The Portfolio</h1>
+      <b>(Scroll to the bottom for a surprise)</b>
+      <p>Hello, I am Anthony Achille, Check out my <a href="https://github.com/Anthony869?tab=repositories">GitHub</a></p>
+      <p></p>
+      <p></p>
+      <button onclick="addRandomGreeting()">Ricky Quotes</button>
       <div id="greeting-container"></div>
+      <p></p>
+      <p></p>
+      <strong>Background:</strong>  
+      <p>I have always been interested in programming and computers in general. Ever since I could remember I wnated to learn
+         more about computer and how they work. Technology has always been interesting and being able to control them with code.
+         As soon as I could take a class about computers in highschool I did, I took Computer Science 3 the first year and continued 
+         with the path after that. Before I could do that at school I went to camps over the summer to learn different languages each time.
```

Comment:
Similar to having an indent size (ie. 2 spaces), you should also have a line length.  80 or 100 is typical, it just makes it easier to read and comprehend, if people do not need to scroll.  GitHub does not make it easy to tell the length, but I can tell these are on the long size.  Decide on a length, and make sure your content adheres to it (note: That sometimes there are exceptions for something that can not be broken into multiple lines).

In general, this is a good time to introduce the idea of a Style Guide.

A Style Guide, is a guide/document that describes the style you will use in writing your code.  The idea is that if a group of 5 people were working on a project together, the code would be a lot more readable if it looked fairly similar.  So, that is the goal.  The good news is that you don't need to write one, you can just adhere to someone elses.

For example, here is Google's:
https://google.github.io/styleguide/

All there code should adhere to this, baring maybe some small exceptions.  Because each language has its own constraints, there is one for each language.  In reality, you are usually scoped to a small set of languages, so it is not that hard to get over time.

**At this time, it is not that important** for you to really understand the style guides, and strictly conform to them.  What is important is just understanding the concept, and that it is important for your code to be consistent.

Reply:
I don't see this changed either, was it?